Task1/aga/IsPrime/IsPrime/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IsPrime
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            IsPrime isPrime = new IsPrime(Convert.ToInt32(Console.ReadLine()));
```

Comment:
What if console input is not a correctly formatted number? This should contain some error handling logic - try catch with some error print.